### DISCUSSION TITLE
Micronaut 4.6.2 vs 4.5.1

### DISCUSSION THREAD

**Discussion by tabhus** [micronaut-projects/micronaut-core#11228] [General]
My  REST service builds and runs fine in 4.5.1 but when I switched to 4.6.2, it  started to give me exceptions: Here are the things I notcied in 4.6.2:

1. After the services is built, it throws an exception, right after the BUILD SUCCESS message, see below:
![image](https://github.com/user-attachments/assets/65a40819-d258-4b13-abf4-a243103b096a)

2. I am using Swagger annotation to generate Swagger doc and noticed it stopped adding the server context path to swagger doc as set in application property file micronaut.server.context-path.
3. When I call any endpoint,  it throws PAGE Not Found exception after successfully executing the endpoint. I get this error after successful exection of my endpoint:

![image](https://github.com/user-attachments/assets/0da0320c-8a6f-4a34-94e3-f21791097e28)

Are there any config change that are needed to migrate to 4.6.2?

Any idea how to fix these issues using 4.6.2?

**Comment by graemerocher** [comment]
are you using Micronaut AOT? //cc @melix

**Reply by tabhus** [reply]
In my POM, I have AOT enabled false...

<micronaut.aot.enabled>false</micronaut.aot.enabled>

**Reply by melix** [reply]
There may be multiple issues here. Issue 1. is actually fixed in the latest test resources version (2.6.2).

**Reply by graemerocher** [reply]
ok so 1 identified issue fixed. @tabhus please create actual issues for the remainder of your problems with steps to reproduce and an example attached. Thank you

**Reply by tabhus** [reply]
Thanks Guys,  I just created an issue.
